backend/battles/views.py

```diff
@@ -1,16 +1,52 @@
 from django.contrib.auth.mixins import LoginRequiredMixin
+from django.http import HttpResponseRedirect
 from django.urls import reverse_lazy
 from django.views.generic import CreateView
 
-from battles.forms import BattleForm
+from battles.forms import BattleForm, TeamForm
+from battles.helpers import save_pokemon_in_team
 from battles.models import Battle
+from services.api import get_pokemon_list
 
 
 class CreateBattleView(LoginRequiredMixin, CreateView):
     template_name = "battles/create_battle.html"
     form_class = BattleForm
     model = Battle
-    success_url = reverse_lazy("home")  # TODO: Change success url to the next step
 
     def get_initial(self):
         return {"user_creator": self.request.user}
+
+    def get_success_url(self):
+        self.success_url = reverse_lazy("battles:create_team", args=[self.object.id])
+        return self.success_url
```

Comment:
If you want you can return the URL directly without storing at some variable. It isn't wrong the way you did, I'm just pointing another option. Feel free to choose what you prefer :)

```suggestion
    def get_success_url(self):
        return reverse_lazy("battles:create_team", args=[self.object.id])
```